pkg/query-service/app/metrics/query_builder_test.go

```diff
@@ -15,19 +16,19 @@ func TestBuildQuery(t *testing.T) {
 			Step:  60,
 			CompositeMetricQuery: &model.CompositeMetricQuery{
 				BuilderQueries: map[string]*model.MetricQuery{
-					"a": {
-						QueryName:         "a",
+					"A": {
```

Comment:
The casing doesn't matter but updated to avoid confusion whit filter query values.